src/OpenSSL/crypto.py

```diff
@@ -1641,10 +1641,11 @@ def set_time(self, vfy_time):
         :param datetime vfy_time: The verification time to set on this store.
         :return: ``None`` if the verification time was successfully set.
         """
+        import calendar
```

Comment:
imports should go at teh top of the file

Reply:
Moved